tests/download_helpers/pypi.py

```diff
@@ -35,14 +36,14 @@ def testGetDownloadURL(self):
     """Tests the GetDownloadURL functions."""
     download_helper = pypi.PyPIDownloadHelper(self._DOWNLOAD_URL)
 
-    expected_download_url = (
-        'https://pypi.python.org/packages/source/d/{0:s}/'
-        '{0:s}-{1:s}.tar.gz').format(self._PROJECT_NAME, self._PROJECT_VERSION)
+    expected_download_url_regexp = re.compile(
+        r'https://files.pythonhosted.org/packages/'
+        r'[\da-f/]+{0:s}-\d{{8}}.tar.gz'.format(self._PROJECT_NAME))
 
     download_url = download_helper.GetDownloadURL(
         self._PROJECT_NAME, self._PROJECT_VERSION)
 
-    self.assertEqual(download_url, expected_download_url)
+    self.assertRegexpMatches(download_url, expected_download_url_regexp)
```

Comment:
self.assertRegexpMatches is not compatible with Python 3. See:
https://docs.python.org/3/library/unittest.html#skipping-tests-and-expected-failures

Hence the string comparison prior to the changes

Reply:
Did you mean to link to https://docs.python.org/3/library/unittest.html#deprecated-aliases ? This test does run with Python 3 (see the Travis test results). I can make this conditional if you'd prefer, but I think it's cleaner to use assertRegexpMatches for now, and change to the other method name once we drop Python 2 compatibility.

Reply:
ack, if it is still supported, we would need a pylint override then for the time being (in case we run pylint with python3), like:
https://github.com/log2timeline/plaso/blob/9e6b728ed6a7da267548a12894fca4a68b97eb9c/tests/cli/psteal_tool.py#L57

Reply:
Done. Perhaps we should also run a "pylint with python3" test to travis, to catch these sort of issues.

Reply:
was thinking the same thing earlier, I've added https://github.com/log2timeline/l2tdevtools/issues/358 as a reminder